src/scripts.js

```diff
@@ -6,39 +6,57 @@ import RecipeRepository from './classes/RecipeRepository';
 import User from './classes/User';
 import recipeData from './data/recipes';
 
-const recipeContainer = document.getElementById('recipe-cards-display-container');
-window.addEventListener('load', () => {
-  buildRecipeRepo();
-  displayAllRecipes();
-})
-
-const buildRecipeRepo = () => {
-  const recipeList = recipeData.reduce((acc, recipe) => {
-    acc.push(new Recipe(
-      recipe.id,
-      recipe.image,
-      recipe.ingredients,
-      recipe.instructions,
-      recipe.name,
-      recipe.tags
-    ));
-    return acc;
-  }, []);
-  return recipeList;
-}
+const recipeRepo = new RecipeRepository(recipeData);
+const recipeContainer = document.getElementById('browse-page');
+const recipeTitle = document.querySelector('#recipe-title');
+const price = document.querySelector('#price');
+const homePage = document.querySelector('.home-page');
```

Comment:
Possibly an opportunity to refactor. Just need to keep a convention for consistency when targeting DOM elements. Let's decide to either use querySelector or getElementById.